### PR TITLE
CF-15  Notification Box 

### PR DESCRIPTION
**Pull request by mrabiabrn**
Proposed changes
----------------

- Built Notification System base (check Notifications component for details)
- Deleted notifications button from the menu, added it in Header
- Created a notifications menu to display ACTIVITIES and FOLLOW REQUESTS

**ACTIVITIES**
  - Displays last 20 activity from activity stream of the user
  - Shows summary and date info for activities
image

**FOLLOW REQUESTS**
  - Displays other users' requests for following the current user
  - If current user's profile is public, this tab will still be shown and no following requests message will be displayed in this tab
  - User can accept following requests by clicking check icon
  image
image

Types of changes
----------------

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

Further comments
----------------

I have not added reject follow request. It is not in our requirements.

### CODE REVIEW

**Review by onurcanavci — APPROVED** [review]
I saw that there are 2 redundant line in .gitignore file, you can delete in next other commit. You do not need to any new commit to delete these files. It looks great :)

File: Columbus/frontend/src/components/Notifications/NotificationMenu.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const { isNotificationsOpen, setIsNotificationsOpen, isNotificationMenuOpen } = props;
+
+const [notifications, setNotifications] = useState([]);
+const [isLoading, setIsLoading] = useState(true);
+
+useEffect(() => {
+
+        USER_SERVICE.GET_NOTIFICATIONS(localStorage.getItem('username'))
+            .then((res) => {
+              setNotifications(res.data.notifications.orderedItems)
```
Comment (onurcanavci) [inline Columbus/frontend/src/components/Notifications/NotificationMenu.js:22]: Sanırım kullanıcının notificationı olmayınca response aşağıdaki gibi oluyor. Bu yüzden de eğer kullanıcıya notification gelmemişse burası patlayabilir :Fyi

Screen Shot 2022-01-01 at 18 07 32

Comment (onurcanavci) [reply Columbus/frontend/src/components/Notifications/NotificationMenu.js:22]: Belki burada `res.data?.notifications` işe yarayabilir ama emin değilim. Diğer türlü notifications objesi olmadığı için crash olması yüksek olası

Comment (mrabiabrn) [reply Columbus/frontend/src/components/Notifications/NotificationMenu.js]: Yes that is true, I will look at it again

Comment (mrabiabrn) [reply Columbus/frontend/src/components/Notifications/NotificationMenu.js]: I wrote this according to the previous endpoint, now I have updated `notifications` to `other_notifications`. It seems to work in this case.

**Review by onurcanavci — APPROVED** [review]

**Review by erencanuysal — APPROVED** [review]

**Pull request merged**